src/pastel/test/lib/ExhaustivenessTest.re

```diff
@@ -8,7 +8,7 @@
 open TestFramework;
 open Pastel;
 
-describe("Pre test runner exhaustiveness test", ({test}) =>
+describe("Pre test runner exhaustiveness test", ({test, _}) =>
```

Comment:
I don't like this change. I can understand the point of warning 9, and why it might be helpful in some cases, but this is explicitly not one of those cases. We don't really care about making sure each field in the record is listed and designed the API in that way on purpose.

Are there examples of this being an important thing that catches bugs? I can understand theoretically how it might come up, but it never seems relevant in practice.

I wonder what some alternatives might be.

Reply:
Yeah, I don't mean to suggest merging this at all, just to make things roll. Please see first two commits and ignore the last two.

As for warning 9 in this repo I think we can just configure dune to the needed set of warnings.

As for warning 9 in general — I commented here
https://github.com/facebookexperimental/reason-native/issues/38#issuecomment-445976340 — don't examples of bugs it prevented right now though. I'll dig something.

Reply:
Yeah sorry - totally understand that. Glad to see this rolling, and this was meant to be a comment more generally about warning 9, not this particular PR. The conversation is probably best for the PR you linked instead of here!